src/components/pages/Theme/Theme.jsx

```diff
@@ -428,6 +430,92 @@ export const Theme = () => {
         </Segment>
       </div>
 
+      <div className='section'>
+        <Segment padded>
+          <Header as='h2' dividing>
+            Table - Semantic UI
```

Comment:
Small detail but we can remove `- Semantic UI`, as I think it's assumed all components on this page are Semantic.

When we add the redux table, that's where we can indicate it's different.

Reply:
Sounds good!